rotkehlchen/exchanges/coinbase.py

```diff
@@ -112,22 +113,27 @@ def trade_from_conversion(trade_a: Dict[str, Any], trade_b: Dict[str, Any]) -> O
 
     timestamp = deserialize_timestamp_from_date(trade_a['updated_at'], 'iso8601', 'coinbase')
     trade_type = deserialize_trade_type('sell')
-    tx_amount = deserialize_asset_amount(trade_b['amount']['amount'])
-    tx_asset = asset_from_coinbase(trade_b['amount']['currency'], time=timestamp)
-    native_amount = deserialize_asset_amount(trade_b['native_amount']['amount'])
-    native_asset = asset_from_coinbase(trade_b['native_amount']['currency'], time=timestamp)
+    tx_amount = AssetAmount(abs(deserialize_asset_amount(trade_a['amount']['amount'])))
+    tx_asset = asset_from_coinbase(trade_a['amount']['currency'], time=timestamp)
+    native_amount = deserialize_asset_amount(trade_b['amount']['amount'])
+    native_asset = asset_from_coinbase(trade_b['amount']['currency'], time=timestamp)
     amount = tx_amount
     # The rate is how much you get/give in quotecurrency if you buy/sell 1 unit of base currency
     rate = Price(native_amount / tx_amount)
 
-    # Obtain fee amount using data from both trades
+    # Obtain fee amount in the native currency using data from both trades
     amount_after_fee = deserialize_asset_amount(trade_b['native_amount']['amount'])
-    amount_before_fee = deserialize_asset_amount(trade_a['amount']['amount'])
-    conversion_fee_amount = amount_after_fee + amount_before_fee
+    amount_before_fee = deserialize_asset_amount(trade_a['native_amount']['amount'])
+    conversion_native_fee_amount = amount_after_fee + amount_before_fee
     # Before this the conversion should be a negative number
-    conversion_fee_amount = abs(conversion_fee_amount)
-
-    fee_amount = deserialize_fee(str(conversion_fee_amount))
+    conversion_native_fee_amount = abs(conversion_native_fee_amount)
```

Comment:
I wrote it like this because my first impression was that this was a bit strange and to make things clear for me I wanted to split this in two lines. If you believe it's better to have in one line I can change it

Reply:
I don't really know what calculations you are doing here. If they solve the problem for the user that's all I care about.

What I do know though is:

bad (too verbose)
```
a = b + c
a = abs(a)
```

good
```
a = abs(b + c)
```